skimage/morphology/greyreconstruct.py

```diff
@@ -42,15 +42,20 @@ def reconstruction(seed, mask, method='dilation', selem=None, offset=None):
         are dilated or eroded.
     mask : ndarray
         The maximum (dilation) / minimum (erosion) allowed value at each pixel.
-    method : {'dilation'|'erosion'}
+    method : {'dilation'|'erosion'}, optional
         Perform reconstruction by dilation or erosion. In dilation (or
         erosion), the seed image is dilated (or eroded) until limited by the
         mask image. For dilation, each seed value must be less than or equal
         to the corresponding mask value; for erosion, the reverse is true.
-    selem : ndarray
+        Default is 'dilation'.
+    selem : ndarray, optional
         The neighborhood expressed as an n-D array of 1's and 0's.
-        Default is the ball of radius 1 according to the maximum norm
-        (i.e. a 3x3 square for 2D images, a 3x3x3 cube for 3D images, etc.)
+        Default is the n-D square of radius equal to 1 (i.e. a 3x3 square
```

Comment:
I'm not sure it's correct to speak about a square of radius 1, how about the square of length 3?